src/features/create-token/CreateTokenModal.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+
+import { BasicModalProps } from '../../lib/types/ui';
+
+import { Modal } from '@zero-tech/zui/components';
+
+import { CreateToken } from './CreateToken';
+
+export interface CreateTokenModalProps extends BasicModalProps {}
+
+export const CreateTokenModal: FC<CreateTokenModalProps> = ({
+	...modalProps
+}) => (
```

Comment:
Don't have to destructure the props here since you're forwarding them all to `Modal`.

```js
export const CreateTokenModal: FC<CreateTokenModalProps> = (props) => (
    <Modal {...props}>
    ...
)
```